### PR TITLE
created google form and spreadsheet for contacts page

### PR DESCRIPTION
**Pull request by courtnaymcewen**
### Impacted areas in the application:  The Contacts page

image

List general components of the application that this PR will affect: 
* 

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts
- [x] I have run the [prettier](https://prettier.io/) command `make pretty`

Note: There are merge conflicts present

### CODE REVIEW

File: frontend/src/components/static_pages/contact.jsx [hunk 1]
```diff
@@ -32,48 +43,18 @@ class Contact extends Component {
...
-          </form>
+        <div className="main-content container col-sm-6 mt-5">
+          <iframe
+            src="https://docs.google.com/forms/d/e/1FAIpQLSe9vicPG5oh6xdZYGLMkksNXQc_6WnjY4GCOZbjcnlVzW0huA/viewform?embedded=true"
+            style={{ width: "100%", height: this.state.iframeHeight }}
+            frameborder="0"
+            marginwidth="0"
+            marginheight="0"
+            onLoad={this.loaded}
+            className="enroll-iframe"
```
Comment (dkaplan88) [inline frontend/src/components/static_pages/contact.jsx]: can we update this to be `contact-iframe` so it is unique

Comment (dkaplan88) [reply frontend/src/components/static_pages/contact.jsx]: lets add add `title='contact-iframe'` as well to get rid of the compiler warning

image

Comment (courtnaymcewen) [reply frontend/src/components/static_pages/contact.jsx]: I added it.

**Comment by courtnaymcewen** [comment]
I have made all of the changes and it is ready to be reviewed.

**Pull request merged**